### ISSUE TITLE
ibus-libpinyin can not export user phrases

### ISSUE THREAD

**Issue by sierxue** [libpinyin/ibus-libpinyin#277]
System: Ubuntu 20.04; English is the default language; ibus-libpinyin is installed by adding input sources through settings.

Problem: After enabling "Remember every input as a phrase" option, ibus-libpinyin **can not** export user phrases.
![Screenshot from 2020-08-29 11-20-53](https://user-images.githubusercontent.com/19348230/91627456-114b7b80-e9ea-11ea-9849-246f05051b4a.png)

**Comment by epico** [comment]
ibus-libpinyin only export the phrase after enabled "Remember every input as a phrase" .

**Comment by sierxue** [comment]
> ibus-libpinyin only export the phrase after enabled "Remember every input as a phrase" .

"Remember every input as a phrase" was enabled, but export didn't work.

**Comment by epico** [comment]
Please try to input some phrases, then check whether the export button works.

**Comment by sierxue** [comment]
> Please try to input some phrases, then check whether the export button works.

The export button works, but it exports to a blank document (the exported file type is txt). I also tried to import a pinyin data file I used on Ubuntu 18.04 first, and then export, but I still got a blank document.

**Comment by epico** [comment]
Please also report this issue to Ubuntu Launchpad, thanks!

**Comment by sierxue** [comment]
> Please also report this issue to Ubuntu Launchpad, thanks!

This has been reported. This url is https://bugs.launchpad.net/ubuntu/+source/ibus-libpinyin/+bug/1893736

**Comment by sierxue** [comment]
@eqico Thank you for your help and your patience!

Finally, I succeeded!

I don't know which steps help to resolve this problem, and just list what I have done:

1. Removed mn_MN and mn_MN.utf8;
2. Run "sudo vim /etc/locale.gen", and uncomment all lines starting with zh_CN, and run "sudo locale-gen";
3. As you instructed, I chose "none" as the Keyboard input method system, and then run "rm ~/.cache/ibus/libpinyin/*", and then reboot.
4. Then I imported a data file used in Ubuntu 18.04 and tested a phrase, which works;
5. Then I try to export, and find it works.
